lib/smart_proxy_dynflow/action/runner.rb

```diff
@@ -14,6 +16,8 @@ def run(event = nil)
           process_external_event(event)
         when ::Dynflow::Action::Cancellable::Cancel
           kill_run
+        when ::Proxy::Dynflow::Action::WithExternalPolling::Poll
+          poll
```

Comment:
we have run defined in the `WithExternalPolling` module don't we want to deffer this to the module?
Our poll is suspending, but the module's `run` calls poll and then suspending, so it might cause confusion and doesn't look too DRY

Reply:
Made those match, module `#poll` does nothing, class `#poll` just polls. In both, `#poll` gets called in `#run` and then the action suspends itself